.github/workflows/client-verify.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.9']
```

Comment:
I'm not the most expert one in python so I would like to know @IceKhan13 opinion here but should we include all our current supported python versions?

Reply:
Yeah, probably makes sense to use all python versions

Reply:
Ideally we want to test all versions, but at least default, which is 3.9 and maximum, which is 3.11 :)